README.md

```diff
@@ -3,12 +3,22 @@
 [![Build Status](https://travis-ci.org/marco-c/crashsimilarity.svg?branch=master)](https://travis-ci.org/marco-c/crashsimilarity)
 [![codecov](https://codecov.io/gh/marco-c/crashsimilarity/branch/master/graph/badge.svg)](https://codecov.io/gh/marco-c/crashsimilarity)
 
-## Tests
+### Description
+Crashsimilarity is a tool to cluster slightly different crash reports, reporting the same problem. It is built for mozilla and support crash reports produced by [breakpad](https://chromium.googlesource.com/breakpad/breakpad/).
```

Comment:
`It is built for mozilla and support crash reports produced by [breakpad](https://chromium.googlesource.com/breakpad/breakpad/).` -> `It supports crashes reported to the Mozilla [Socorro crash reporting system](link to socorro home page)`

No need to mention Breakpad, as the project doesn't really depend on the particular software that generates the reports.